test/e2e/e2e_test.go

```diff
@@ -7,20 +7,21 @@ import (
 	"testing"
 	"time"
 
+	"github.com/openshift/service-ca-operator/pkg/controller/api"
+	"github.com/openshift/service-ca-operator/pkg/operator/operatorclient"
+
 	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/clientcmd"
-
-	"github.com/openshift/service-ca-operator/pkg/controller/api"
 )
 
 const (
-	serviceCAOperatorNamespace   = "openshift-service-ca-operator"
-	serviceCAControllerNamespace = "openshift-service-cert-signer"
-	serviceCAOperatorPodPrefix   = "service-ca-operator"
+	serviceCAOperatorNamespace   = operatorclient.OperatorNamespace
+	serviceCAOperatorPodPrefix   = operatorclient.OperatorNamespace // Same as operator namespace
+	serviceCAControllerNamespace = operatorclient.TargetNamespace
 	apiInjectorPodPrefix         = "apiservice-cabundle-injector"
 	configMapInjectorPodPrefix   = "configmap-cabundle-injector"
 	caControllerPodPrefix        = "service-serving-cert-signer"
```

Comment:
Optional: we can use these too. https://github.com/openshift/service-ca-operator/blob/a92546fbaa0c89374c4b4981f57e5b44aae88590/pkg/controller/api/resourcenames.go#L22-L25

Reply:
done